microprofile.jdt/org.eclipse.lsp4mp.jdt.core/src/main/java/org/eclipse/lsp4mp/commons/utils/JSONUtility.java

```diff
@@ -0,0 +1,78 @@
+/*******************************************************************************
+* Copyright (c) 2019 Red Hat Inc. and others.
```

Comment:
2022

Reply:
This isn't new, I just copied it over from microprofile.ls